lib/imageResize.ts

```diff
@@ -63,14 +101,33 @@ export const ImageResize = Image.extend({
         $leftController.setAttribute('style', iconStyle);
         $leftController.addEventListener('mouseover', controllerMouseOver);
         $leftController.addEventListener('mouseout', controllerMouseOut);
+        $leftController.addEventListener('click', () => {
+          if (!inline) {
+            $container.setAttribute('style', `display: inline-block; margin: 0 auto 0 0;`);
+          } else {
+            $wrapper.setAttribute(
+              'style',
+              `display: ${display}; float: left; padding-right: 8px;`
```

Comment:
The display variant is always `inline-block`, right?
If so, we don't need to use the variant, just like in the $rightController code.